src/app/component/ImageUpload/ImageUpload.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render, cleanup, fireEvent } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+import { ImageUpload } from '.';
+
+describe('ImageUpload Component', () => {
+  afterEach(cleanup);
+
+  global.URL.createObjectURL = jest.fn(); // Mock
+
+  it('renders camera icon and two buttons', () => {
+    const { getByText, getByTestId } = render(<ImageUpload />);
+    expect(getByTestId('cameraIcon')).toBeTruthy();
```

Comment:
Minor: I think explicit assertion i.e. toBe(true) is better than Truthy tests as the latter sometimes give false positives i.e. empty objects evaluate to true

Reply:
Using toBe(true) will not be the best option here since toBe will compare it to a string instead checking whether it's available. 
![image](https://user-images.githubusercontent.com/11330782/77334492-c764b800-6d1c-11ea-947e-b1fc0e0e9252.png)

The best matcher to use is `toBeInTheDocument`